docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -369,7 +369,7 @@ Save this file in `src/main/docker/Dockerfile.multistage` as it is not included
 
 [WARNING]
 ====
-Replace the -java8 in the base image with -java11 if you want to use Java 11.
+Replace `-java11` in the base image with `-java8` if you want to use Java 8.
```

Comment:
This doesn't really make sense now that `-java8` is gone (and `-java11` does not exist AFAICS).